tests/target/cargo-toml/false/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+# rustfmt-format_cargo_toml: false
+
+  # comment before [[bin]], not necessarily at the beginning
+
+
+    [[bin]]    # comment  after [[bin]]
+        "aa" = 1     # comment for "aa"    
+        'bb' = 2 
+        "啊"=1
+
+         # comment before package  
+
+[package]
+    version = 1
+    description = "a\nb\nhaha"
+    name = 3
+
+   # comment 1  for arr1 line1
+   
+# comment 1 for arr1 line 2
+    arr1 = [1,
+        2,3]  
+
+# comment 2   for arr2   
+
+    arr2 = ["11111111111111111111111111111111111111111111111111111111111111111111111111111111","1111111111111111111111111111111111111111111111111111111111111111111111111111111"]
+
+   # comment before [[dependencies]], not after [package]  
+
+
+[dependencies]
+        extremely_long_crate_name_goes_here = {path = "extremely_long_path_name_goes_right_here",version = "4.5.6"}
+
+        
+        crate1 = {            path = "crate1",version = "1.2.3"                   }
+   
+# comment before the second [[bin]]
+
+[[bin]]
+        d = "git-rustfmt"
+        c = "src/git-rustfmt/main.rs"
+
+
+
+
+    #   comment at the end of the file   
+
+
+    
```

Comment:
Same minor nit about newlines. Also, can we move the tests to `tests/target/configs/format_cargo_toml/` and `tests/source/configs/format_cargo_toml/` similar to how we've got other tests for other config options.

Reply:
done